Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,45 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+
+
+
+// Function clean and normalize the input string (remove punctuations and convert to all lower case)
+function removePunctuations(inputStr) {
+  return inputStr
+    .replace(/[.,!?]/g, '')
+    .toLowerCase();
+}
+// Function to split a string into words
+function splitToWords(removePunctuations) {
+  return removePunctuations.split(/\s+/);
+}
```

Comment:
As this function works for any string, you can use a more generic name for the parameter.